text/0000-cargo-features2.md

```diff
@@ -0,0 +1,514 @@
+- Feature Name: `cargo-features2`
+- Start Date: 2020-05-09
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+
+This RFC is to gather final feedback on stabilizing the new feature resolver
+in Cargo. This new feature resolver introduces a new algorithm for computing
+[package features][docs-old-features] that helps to avoid some unwanted
+unification that happens in the current resolver. This also includes some
+changes in how features are enabled on the command-line.
+
+These changes have already been implemented and are available on the nightly
+channel as an unstable feature. See the [unstable feature docs] for
+information on how to test out the new resolver, and the [unstable package
+flags] for information on the new flag behavior.
```

Comment:
Maybe the fact that there are still many problems with features and that this will not fix all of them could be mentioned.